assets/training/finetune_acft_image/components/finetune/mmt/spec.yaml

```diff
@@ -0,0 +1,341 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+type: command
+
+version: 0.0.1
+name: mmtracking_video_multi_object_tracking_finetune
+display_name: Video Multi Object Tracking MMTracking Model Finetune
+description: Component to finetune MMTracking models for video multi-object tracking task.
+
+is_deterministic: false
+
+environment: azureml://registries/azureml/environments/acft-mmtracking-video-gpu/versions/1
+
+code: ../../../src/finetune
+
+distribution:
+  type: pytorch
+
+inputs:
+
+  # component input: model path
+  model_path:
+    type: uri_folder
+    optional: false
+    description: Output folder of model selector containing model metadata like config, checkpoints, tokenizer config.
+
+  # component input: training mltable
+  training_data:
+    type: mltable
+    optional: false
+    description: Path to the mltable of the training dataset.
+
+  # optional component input: validation mltable
+  validation_data:
+    type: mltable
+    optional: true
+    description: Path to the mltable of the validation dataset.
+
+  image_width:
+    type: integer
+    default: -1
+    optional: true
+    description: image width that is input to the network. Default 
+                 is -1 which means it would be overwritten by image_scale in model config. 
+
+  image_height:
+    type: integer
+    default: -1
+    optional: true
+    description: image height that is input to the network. Default 
+                 is -1 which means it would be overwritten by image_scale in model config.
```

Comment:
extra spaces - can we please clean those up?

Reply:
all updated accordingly. no more new lines !! thanks Rupal!